server/models/issue.js

```diff
@@ -0,0 +1,61 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+const IssueSchema = new Schema(
+  {
+    description: {
+      type: String,
+      trim: true,
+      required: true,
```

Comment:
desc may not be requoired

Reply:
It is required. Description is the actual issue. I think I should change the name and add a new description field.

Reply:
Removed ambiguity now.